app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -40,13 +61,14 @@
 
     private WebView mWebView;
     private Uri mUri;
-    int mPage;
-    int mNumPages;
+    public int mPage;
```

Comment:
No need to mark these as public, the default without a visibility marker is package-private which is a bit nicer than public although here it doesn't really make any difference.